### DISCUSSION TITLE
Built-in it dictionary location

### DISCUSSION THREAD

**Discussion by nameOfUser0236** [streetsidesoftware/vscode-spell-checker#1222] [Q&A] [answered]
where are the default dictionaries located (eg. softwareTerms)?
I can't seem to locate them.

**Comment by Jason3S** [comment, marked as answer]
They can be found at [cspell-dicts](https://github.com/streetsidesoftware/cspell-dicts)

**Reply by nameOfUser0236** [reply]
Thanks!!

**Reply by nameOfUser0236** [reply]
Github is being weird and won't let me accept your answer. Sorry.
